packages/coinstac-ui/app/render/components/consortium/consortium-results.js

```diff
@@ -2,46 +2,52 @@ import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { Alert } from 'react-bootstrap';
-import { setExpandedResult } from '../../state/ducks/consortia-page';
+import { setExpandedResults } from '../../state/ducks/consortia-page';
 
 import ConsortiumResult from './consortium-result';
 
 class ConsortiumResults extends Component {
   constructor(props) {
     super(props);
 
+    this.state = { wentToResult: false };
     this.goToResults = this.goToResults.bind(this);
```

Comment:
*wentToResult* is a flag on the current state to prevent the page from jumping each time a new render cycle is encountered. Just indicates that the *goToResult()* function has already been called.